android/templates/module/kotlin/template/android/src/{{ModuleIdAsFolder}}/ExampleProxy.kt.ejs

```diff
@@ -0,0 +1,86 @@
+/**
+ * This file was auto-generated by the Titanium Module SDK helper for Android
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2020 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ *
+ */
+
+package <%- moduleId %>
+
+import android.app.Activity
+import org.appcelerator.kroll.KrollDict
+import org.appcelerator.kroll.annotations.Kroll.*
```

Comment:
I don't think we should do a wildcard import, which means prefixing everything with `Kroll.` down below.